src/PhpCs/FiveLab/Sniffs/Strings/AsciiSniff.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the FiveLab CiRules package
+ *
+ * (c) FiveLab
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code
+ */
+
+declare(strict_types = 1);
+
+namespace FiveLab\Component\CiRules\PhpCs\FiveLab\Sniffs\Strings;
+
+use FiveLab\Component\CiRules\PhpCs\FiveLab\ErrorCodes;
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class AsciiSniff implements Sniff
+{
+    public function register(): array
+    {
+        return [
+            T_VARIABLE,
+            T_CLASS,
+            T_INTERFACE,
+            T_FUNCTION,
+            T_STRING,
+            T_CONSTANT_ENCAPSED_STRING,
+            T_DOUBLE_QUOTED_STRING,
+            T_COMMENT,
+            T_DOC_COMMENT_STRING,
+        ];
+    }
+
+    public function process(File $phpcsFile, $stackPtr): void
+    {
+        $tokens = $phpcsFile->getTokens();
+        $content = $tokens[$stackPtr]['content'];
+        $forbiddenSymbols = [];
+
+        foreach (mb_str_split($content) as $char) {
```

Comment:
Should be `\` before call function in global namespace. And please check this moment below.